src/main.ts

```diff
@@ -1424,7 +1476,26 @@ function generateUnwrap(fullProtoTypeName: string): Code[] {
   }
 
   if (isAnyValueTypeName(fullProtoTypeName)) {
-    chunks.push(code`unwrap(message: Value): string | number | boolean | Object | null | Array<any> | undefined {
+    if (ctx.options.oneof === OneofOption.UNIONS) {
+      chunks.push(code`unwrap(message: Value): string | number | boolean | Object | null | Array<any> | undefined {
+        if (message.kind?.$case === 'nullValue') {
```

Comment:
Ah sure, this makes sense...

Fwiw I was looking for a `struct.ts` that had this code in it, but don't see it ... looks like you did change the oneof-union test case to use a struct, but maybe the new `google/protobuf/struct.ts` just didn't get checked in as part of the PR?

Reply:
Great catch! Wow :-) Just pushed it.